src/libsyncengine/update_detection/file_system_observer/localfilesystemobserverworker.h

```diff
@@ -37,7 +37,7 @@ class LocalFileSystemObserverWorker : public FileSystemObserverWorker {
 
     protected:
         virtual void execute() override;
-        ExitInfo exploreDir(const SyncPath &absoluteParentDirPath);
+        ExitInfo exploreDir(const SyncPath &absoluteParentDirPath, bool changeDetected = false);
```

Comment:
Could you please document this option?

Reply:
Changed to a more explicit name here: [36c844d](https://github.com/Infomaniak/desktop-kDrive/pull/403/commits/36c844d03aca707b493d8b2b0510e39761a25439)